src/simulation/m_global_parameters.fpp

```diff
@@ -491,6 +491,7 @@ contains
             mono(j)%support = 1
             mono(j)%foc_length = dflt_real
             mono(j)%aperture = dflt_real
+            mono(j)%support_width = 2.5d0
```

Comment:
Where does 2.5 come from?

Reply:
It was the hard-coded default for monopole.

Reply:
please add a comment for this line. also, why is this a new line? shouldn't the support width have been already present in the code?

Reply:
It's not in the code yet. The new input is the number of characteristic cells the support spans. It uses the new input to calculate the 'sig' which is the actual width of the support.

Reply:
Sounds good to me.